tasks/bundle-js.js

```diff
@@ -28,6 +30,19 @@ function patchFirefoxJS(/** @type {string} */code) {
     return code;
 }
 
+function patchMV3JS(/** @type {string} */code) {
+    // MV3 moves a few APIs around
+    code = replace(code, 'chrome.browserAction.setIcon', 'chrome.action.setIcon');
+    code = replace(code, 'chrome.browserAction.setBadgeBackgroundColor', 'chrome.action.setBadgeBackgroundColor');
+    code = replace(code, 'chrome.browserAction.setBadgeText', 'chrome.action.setBadgeText');
+
+    // Background serivice workers do not have access to "foreground" APIs like chrome.i18n
+    code = replace(code, 'chrome.i18n.getUILanguage()', "navigator.language.split('-')[0]");
```

Comment:
Should I move this to `src/utils/locales.ts`?

Reply:
Let's do this